peekingduck/pipeline/nodes/output/screen.py

```diff
@@ -49,15 +51,33 @@ def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
         cv2.namedWindow(self.window_name, cv2.WINDOW_NORMAL)
         cv2.moveWindow(self.window_name, self.window_loc["x"], self.window_loc["y"])
+        self.aspect_ratio_thres = 5e-3
```

Comment:
Use `threshold`, don't have to abbreviate

Reply:
Amended